.github/workflows/dbt_slim_ci.yml

```diff
@@ -59,3 +59,8 @@ jobs:
       - name: dbt test incremental model(s) if applicable
         if: env.INC_MODEL_COUNT > 0
         run: "dbt test --select state:modified,config.materialized:incremental --defer --state ."
+
+      - name: Run DuneSQL Check
+        if: env.INC_MODEL_COUNT > 0
+        run: "/runner/dunesql_check.py --schema test_schema --pr_schema git_$GIT_SHA"
+        continue-on-error: true
```

Comment:
assume this means it won't fail the CI test, if it fails the dunesql check? that's probably best in the early stages?

Reply:
sounds good 👍

Reply:
oh wow I love this solution. 

so much cleaner to use the schema instead of the manifest and I appreciate the continue on error as a conservative way of adding this.